apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js

```diff
@@ -0,0 +1,85 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { ApplicationsHomePage } from '../../page_objects/applicationsHomePage';
+import { CreateCasePage } from '../../page_objects/createCasePage';
+import { SearchResultsPage } from '../../page_objects/searchResultsPage';
+import { ProjectTeamPage } from '../../page_objects/projectTeamPage';
+import { projectInformation } from '../../support/utils/createProjectInformation';
+
+const createCasePage = new CreateCasePage();
+const projectTeamPage = new ProjectTeamPage();
+const applicationsHomePage = new ApplicationsHomePage();
+const searchResultsPage = new SearchResultsPage();
+const { applications: applicationUsers } = users;
+
+describe('Create case ', () => {
```

Comment:
better name this: `Case Team` or something like that

Reply:
Sure